### PR TITLE
fonttools: migrate to multi-Python port

### PR DESCRIPTION
**Pull request by amake**
#### Description

Per [this request](https://trac.macports.org/ticket/63464) this PR obsoletes the fonttools port in favor of py-fonttools + fonttools_select.

This is my first *_select port, so I hope I got it right.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on
macOS 11.5.2 20G95 x86_64
Xcode 12.5.1 12E507

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Review by catap — APPROVED** [review]
works for me, thanks!

File: python/py-fonttools/Portfile [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           select 1.0
+
+name                py-fonttools
+version             4.26.2
```
Comment (reneeotten) [inline python/py-fonttools/Portfile:8]: according to upstream, this version doesn't support Python 2.7, so i you still need the PY27 subport that one should likely be restricted to the 3.x branch.

Comment (amake) [reply python/py-fonttools/Portfile:8]: It already is: https://github.com/macports/macports-ports/pull/12150/files#diff-75dfe2fa780c961bc28af98945fd6ef85533318499239b8aec011578870c2e36R37-R43

Comment (reneeotten) [reply python/py-fonttools/Portfile:8]: oh, I am sorry @amake ! I should have some coffee first ;)

File: print/fonttools/Portfile [hunk 2]
```diff
@@ -1,81 +1,11 @@
...
-
-post-destroot {
-    foreach manfile [glob -tails -directory ${destroot}${python.prefix}/share/man/man1 *] {
-        ln -s ${python.prefix}/share/man/man1/${manfile} \
-            ${destroot}${prefix}/share/man/man1/${manfile}
-    }
-}
+replaced_by         py39-fonttools
+version             4.26.2
+revision            1
```
[the remark this answers is not in the document]
Comment (amake) [reply print/fonttools/Portfile:12]: Good idea, thanks. Done in https://github.com/macports/macports-ports/commit/81464d587cae8d5d625abbde590ee31d1b893670

**Review by reneeotten — APPROVED** [review]

**Pull request merged**